plugins/woocommerce/src/Blocks/BlockTypes/AccordionGroup.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+/**
+ * AccordionGroup class.
+ */
+class AccordionGroup extends AbstractBlock {
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'accordion-group';
+
+	/**
+	 * Include and render the block.
+	 *
+	 * @param array    $attributes Block attributes. Default empty array.
+	 * @param string   $content    Block content. Default empty string.
+	 * @param WP_Block $block      Block instance.
+	 * @return string Rendered block type output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		if ( ! $content ) {
+			return $content;
+		}
+	
+		$suffix = wp_scripts_get_suffix();
+		if ( defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ) {
+			$module_url = gutenberg_url( '/build-module/block-library/accordion-group/view.min.js' );
+		}
+	
+		$script_data = $this->asset_api->get_script_data( $this->asset_api->get_block_asset_build_path( $this->block_name . '-frontend' ), array(  ) );
+		wp_register_script(
+			'wc-accordion-group-frontend',
+			$script_data['src'],
+			$script_data['dependencies'],
+			$script_data['version']
+		);
+	
+		wp_enqueue_script( 'wc-accordion-group-frontend' );
```

Comment:
I think that we can avoid this, if we follow the approach above.

Reply:
Done.